.dependabot/config.yml

```diff
@@ -0,0 +1,8 @@
+version: 1
+update_configs:
+  - package_manager: "javascript"
+    directory: "/"
```

Comment:
Question about all of the specified directories (both this config and github one) - I guess it includes all of the files and we don't need to specify a directory? (unless we have some parts of app with separate package.json that we don't want to bump )

Reply:
I believe yes, that specifies the location of the package.json file